configs/pontoon.toml

```diff
@@ -124,6 +124,26 @@ locales = [
 [[paths]]
     reference = "en/firefox/*.ftl"
     l10n = "{locale}/firefox/*.ftl"
+[[paths]]
+    reference = "en/firefox/browsers/*.ftl"
+    l10n = "{locale}/firefox/browsers/*.ftl"
+[[paths]]
+    reference = "en/firefox/browsers/compare/**/*.ftl"
+    l10n = "{locale}/firefox/browsers/compare/**/*.ftl"
+[[paths]]
+    reference = "en/firefox/browsers/history/**/*.ftl"
+    l10n = "{locale}/firefox/browsers/history/**/*.ftl"
+    locales = [
```

Comment:
I thought the page is open to all locales, not limited to these listed here.

Reply:
I'm going off this config: https://github.com/mozilla-l10n/langchecker/blob/master/app/config/sources.inc.php#L502-L509

Happy to open it up to everyone though, as that means less config? :)

Reply:
Glad you are using the .langchecker config file as reference. The pages with opt-in option were initially opened to de,fr locales only so no to overwhelm the communities by these pages with tons of content. They did so by filing a bug. The config for Fluent limits to only those locales, which begs the question, if a community wants to opt in, is there a way to do it? Do I file an issue in Bedrock to make a request?  Back to this page, it will be expanded to broader locale list once we shift to vendor service covering a standard set of languages.

Reply:
We’re using the Fluent project config format, so I’m not actually sure of the best way to add support opt-in locales. Feels a bit like this is out of bedrock’s remit?

For this pull request there are currently two options: we use the subset of locales, or open up to everyone. Which would you prefer here and now?